### DISCUSSION TITLE
Welcome to FlexBazaar Discussions!

### DISCUSSION THREAD

**Discussion by ChristofBecu** [sigarettenenkoffie/FlexBazaar#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChristofBecu** [comment]
@HannesDerynck 
Heb de Vue aan de praat gekeregen, damn CORS :D 
communiceert met de api en toont response.
Wel gepushed naar de dev

**Reply by HannesDerynck** [reply]
nice! ik ben es bezig aan het kijken voor die identity server op te zetten. Kben hier wel voor op een aparte branch aan het werken. Ik heb hier al zitten kijken waar dat kanban board zich bevindt op github, maar ik heb er geen idee meer van. Weet jij het?

**Reply by ChristofBecu** [reply]
Aha, github copilot to the rescue 

![Screenshot_20240523-193032.png](https://github.com/sigarettenenkoffie/FlexBazaar/assets/16795074/4b44b53d-8681-4e25-8540-017c72c23524)

**Reply by ChristofBecu** [reply]
Ma boards zien nie supported in mobile github

**Comment by ChristofBecu** [comment]
@HannesDerynck yoo, wa is nu weer die Core layer? 
Infrastructure & Domain snap ik nog wel, maar hoe meer dak over die Core zie, hoe minder dak er van snap.
Is da ton hetzelfde als Application Layer daj ook soms ziet waar da de bussines logic in zit (services, commands,...)
Ken de indruk da verschillende bronnen elkaar tegenspreken in ulder uitleg... :(

**Reply by HannesDerynck** [reply]
Ja daar zou normaal de business logic in moeten zitten. En dan zoud je API eigenlijk constant die Core layer moeten aanspreken.

**Comment by ChristofBecu** [comment]
@HannesDerynck 

Das die repo dak van't weekend gevonden heb.
[booking-modular-monolith](https://github.com/meysamhadeli/booking-modular-monolith)

Ge moe es da stukje lezen over vertical slice architecture en dan es naar de source kijken. De API runt de modules (slices)  :-o
ook da stuk over feature folder structure lijkt meer dan interessant in da opzicht...
